wp-seo.php

```diff
@@ -1,5 +1,10 @@
 <?php
-/*
+/**
+ * @package    WPSEO
+ * @subpackage Main
+ */
+
+/**
```

Comment:
Yes, in line with PSR-5 everything related should be docblock formatted, see [4. Basic Principles](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#4-basic-principles)